buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    mavenCentral()
+    gradlePluginPortal()
+}
+
+dependencies {
+    implementation("io.cloudflight.autoconfigure.swagger-api-configure:io.cloudflight.autoconfigure.swagger-api-configure.gradle.plugin:1.1.1")
```

Comment:
if we really want a "clean gradle" we should move all those version definitions to the version catalog